src/Control/Optics/Linear/Internal.hs

```diff
@@ -66,11 +75,29 @@ get :: Optic_ (Kleisli (Const a)) a b s t -> s -> a
 get l = gets l P.id
 
 gets :: Optic_ (Kleisli (Const r)) a b s t -> (a -> r) -> s -> r
-gets (Optical l) f s = getConst (runKleisli (l (Kleisli (Const P.. f))) s)
+gets (Optical l) f s = getConst' (runKleisli (l (Kleisli (Const P.. f))) s)
 
 set :: Optic_ (->) a b s t -> b -> s -> t
 set (Optical l) x = l (const x)
 
+-- get can't return a linear arrow, so `withIso swap const` needs to be used
+-- instead of `get swap`
```

Comment:
I'm not sure what his means. You may want to expand on it until it's crystal clear.

> The lens library defines @match@ as … . But this is not possible here, because … .

Etc…

Reply:
Ah this was much more of a throwaway comment: I just meant that in the two definitions `match` and `match'` given here, both need a `swap` to convert `Either a t` to `Either t a`. To do this, one could use `get swap` to swap it, but `get swap` isn't linear so wouldn't work for `match`. 

In hindsight it would probably be better to write
```haskell
match' (Optical l) = forget (withIso swap (\x _ -> x)) P.. runKleisli (l (Kleisli Left))
```

Reply:
I've changed the definition and removed this comment.